Source/VMware.vSphereDSC/VMware.vSphereDSC.build.ps1

```diff
@@ -91,6 +204,24 @@ foreach ($folder in $script:ImportFolders) {
     }
 }
 
+# Add License to psm1 file.
+"<#" | Out-File -FilePath $script:PsmPath -Encoding Default
+$script:LicenseFileContent | ForEach-Object { $_ | Out-File -FilePath $script:PsmPath -Encoding Default -Append }
+"#>" + [System.Environment]::NewLine | Out-File -FilePath $script:PsmPath -Encoding Default -Append
+
+# Add helper module to psm1 file.
+"Using module '.\VMware.vSphereDSC.Helper.psm1'" | Out-File -FilePath $script:PsmPath -Encoding Default -Append
+
+# Updating VMware.vSphereDSC.psm1 content with enums.
+Update-ContentOfModuleFile -Folder $script:EnumsFolder
+
+# Updating VMware.vSphereDSC.psm1 content with classes.
+$orderedClassesFiles = Get-OrderedFiles -Files $script:ClassesFiles
```

Comment:
Add a comment why the file has to be ordered, and what is this kind of ordering.